testing/bootstrap.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# This script is used to bootstrap our prow jobs.
+# The point of this script is to check out the kubeflow/kubeflow repo
+# at the commit corresponding to the Prow job. We can then
+# invoke the launcher script at that commit to submit and
+# monitor an Argo workflow
```

Comment:
and what's an argo workflow.  (1 sentence)

Reply:
Oh, I need to remove that. That's some copy pasta.